lxd/vm_qemu.go

```diff
@@ -1022,28 +1040,66 @@ bootindex = "1"
 	return nil
 }
 
+// addConfDriveConfig adds the qemu config required for adding the config drive.
 func (vm *vmQemu) addConfDriveConfig(sb *strings.Builder) {
 	sb.WriteString(fmt.Sprintf(`
 # Config drive
-[fsdev "qemu_config"]
+[fsdev "lxd_config"]
 fsdriver = "local"
 security_model = "none"
 readonly = "on"
 path = "%s"
 
-[device "dev-qemu_config"]
+[device "dev-lxd_config"]
 driver = "virtio-9p-pci"
-fsdev = "qemu_config"
+fsdev = "lxd_config"
 mount_tag = "config"
 `, filepath.Join(vm.Path(), "config")))
 
 	return
 }
 
-func (vm *vmQemu) addNetConfig(sb *strings.Builder, tapDev map[string]string) {
+// addDriveConfig adds the qemu config required for adding a supplementary drive.
+func (vm *vmQemu) addDriveConfig(sb *strings.Builder, driveIndex int, driveConf device.MountEntryItem) {
+	driveName := fmt.Sprintf(driveConf.TargetPath)
+
+	sb.WriteString(fmt.Sprintf(`
+# %s drive
+[drive "lxd_disk_%s"]
```

Comment:
Per above, should just be `lxd_%s`  as we're encoding the name of the device in there and those are unique.
We just need to make sure that any device which isn't listed in ExpandedDevices (internal ones) are `qemu_` instead.

Reply:
@stgraber will do